### PR TITLE
Started reworking program page

### LINKED ISSUE #99: Show event day theme only for talks

### PR DESCRIPTION
**Pull request by Naapperas**
Closes #99

### CODE REVIEW

**Comment by Naapperas** [comment]
This PR is waiting for #105

File: app/Traits/HasCV.php [hunk 1]
```diff
@@ -19,6 +19,8 @@ public function updateCV(UploadedFile $cv, $storagePath = 'cvs')
         tap($this->cv_path, function ($previous) use ($cv, $storagePath) {
             if ($previous) {
                 Storage::disk($this->CVDisk())->delete($previous);
+            } else {
+                // Give points for CV submission
```
Comment (pedronunes19) [inline app/Traits/HasCV.php:23]: So we're still not allowing users to delete their CV, right?

File: resources/js/Components/Navbar.vue [hunk 2]
```diff
@@ -91,7 +95,10 @@ const isAdmin = computed(() => {
                         <DropdownLink
                             :href="route('competition.show', { competition })"
                         >
-                            {{ competition.name }}
+                            {{
+                                competition.name ??
+                                `Competition ${competition.id}`
+                            }}
```
Comment (pedronunes19) [inline resources/js/Components/Navbar.vue]: why are we allowing null competition names?

**Comment by pedronunes19** [comment]
How are we dealing with other events like the panel, for example?

**Comment by Naapperas** [comment]
> How are we dealing with other events like the panel, for example?

That could be another event type. CC @ttoino

File: resources/js/Components/Navbar.vue (hunk 2 above)
Comment (Naapperas) [reply resources/js/Components/Navbar.vue]: We are not, I don't know why I put that there but it's fixed now.

File: app/Traits/HasCV.php (hunk 1 above)
Comment (Naapperas) [reply app/Traits/HasCV.php:23]: For now at least.

**Comment by ttoino** [comment]
> > How are we dealing with other events like the panel, for example?
> 
> That could be another event type. CC @ttoino

It could be another type or it could just be a talk, we support multiple speakers already.

**Review by toni-santos — APPROVED** [review]

**Pull request merged**